CHANGES.md

```diff
@@ -23,6 +23,7 @@
   This change allows users to push the tag manually to avoid using that code. (#219, @Julow)
 - Recursively exclude all `.git`/`.hg` files and folders from the distrib
   tarball (#300, @NathanReb)
+- Make the automatic dune-release workflow to stop if a step doesn't return 0 (#332, @gpetiot)
```

Comment:
How about clarifying that with `to stop if a step exits with a non-zero code`? I think the `return 0` part might be a bit confusing whereas a slightly different phrasing makes it clear that it means one of the individual commands failed.

What do you think?

Reply:
I agree